scripts/plotting/tem.py

```diff
@@ -71,26 +68,34 @@ def tem(adf):
     redo_plot = adf.get_basic_info('redo_plot')
     print(f"\t NOTE: redo_plot is set to {redo_plot}")
     #-----------------------------------------
+    
+    #Initialize list of input TEM file locations
+    tem_locs = []
 
-    #Grab TEM diagnostics options
-    tem_opts = adf.read_config_var("tem_info")
-
-    if not tem_opts:
-        print("\n  No TEM options provided, skipping TEM plots." \
-        "\nSee documentation or config_cam_baseline_example.yaml for options to add to configuration file.")
-        return
+    #Extract TEM file save locations
+    tem_case_locs = adf.get_cam_info("cam_tem_loc",required=True)
+    tem_base_loc = adf.get_baseline_info("cam_tem_loc")
 
-    #Location of saved TEM netCDF files
-    tem_loc = tem_opts.get("tem_loc")
+    #THSI IS IF IT IS MISSING!
+    #if tem_case_locs is not None:
+    #    if (len(tem_case_locs) == 1) and (any(tem_case_locs) is None):
+    #        tem_case_locs = None
```

Comment:
I would vote to either un-comment or remove this code block, depending on whether you think this check is needed (or not).